src/sentry/api/endpoints/event_ai_suggested_fix.py

```diff
@@ -264,28 +250,18 @@ def describe_event_for_ai(event, model):
     return data
 
 
-def suggest_fix(event_data, model=settings.SENTRY_AI_SUGGESTED_FIX_MODEL, stream=False):
+def suggest_fix(event_data, model=settings.SENTRY_AI_SUGGESTED_FIX_MODEL):
     """Runs an OpenAI request to suggest a fix."""
     prompt = PROMPT.replace("___FUN_PROMPT___", random.choice(FUN_PROMPT_CHOICES))
     event_info = describe_event_for_ai(event_data, model=model)
```

Comment:
Do we want to use the model configured in the usecase config instead of `settings.SENTRY_AI_SUGGESTED_FIX_MODEL` now?

Reply:
ah good point, this parameter shouldn't be used anymore